.github/workflows/spelling2.yml

```diff
@@ -3,20 +3,70 @@ name: Spell checking
 on:
```

Comment:
This is basically:
https://github.com/check-spelling/spell-check-this/blob/prerelease/.github/workflows/spelling.yml
with a pinned release of `v0.0.20-alpha3`. I'm close to releasing a non-alpha but am still working through a couple of minor things. (And I'm debating the version number....)